src/v/config/configuration.cc

```diff
@@ -396,6 +396,12 @@ configuration::configuration()
       "cache",
       required::no,
       60s)
+  , max_compacted_log_segment_size(
```

Comment:
Would be good to add this option to the `docs/www/advanced-configuration.md` with a a sentence of context